cmd/mt-kafka-mdm-report-out-of-order/flags.go

```diff
@@ -0,0 +1,146 @@
+	fmt.Fprintln(os.Stderr, "  ...")
+	fmt.Fprintln(os.Stderr, "  out-of-order metric points grouped by tag=\"fruit\":")
+	fmt.Fprintln(os.Stderr, "  out-of-order metric points for tag=\"fruit\" value=\"apple\" count=80 percentGroup=5.856515 percentClass=2.062919 percentTotal=0.002951")
+	fmt.Fprintln(os.Stderr, "  out-of-order metric points for tag=\"fruit\" value=\"orange\" count=2912 percentGroup=0.306267 percentClass=75.090253 percentTotal=0.107422")
+	fmt.Fprintln(os.Stderr, "  ...")
+	fmt.Fprintln(os.Stderr, "  total duplicate metric points count=12760")
+	fmt.Fprintln(os.Stderr, "  duplicate metric points grouped by name:")
+	fmt.Fprintln(os.Stderr, "  duplicate metric points for name=\"fruit.width\" count=105 percentGroup=19.266055 percentClass=0.760704 percentTotal=0.003397")
+	fmt.Fprintln(os.Stderr, "  duplicate metric points for name=\"fruit.length\" count=123 percentGroup=15.688776 percentClass=0.891111 percentTotal=0.003979")
+	fmt.Fprintln(os.Stderr, "  ...")
+	fmt.Fprintln(os.Stderr, "  duplicate metric points grouped by tag=\"fruit\":")
+	fmt.Fprintln(os.Stderr, "  duplicate metric points for tag=\"fruit\" value=\"banana\" count=4002 percentGroup=17.201066 percentClass=31.363636 percentTotal=0.147631")
+	fmt.Fprintln(os.Stderr, "  duplicate metric points for tag=\"fruit\" value=\"orange\" count=4796 percentGroup=0.504415 percentClass=37.586207 percentTotal=0.176922")
+	fmt.Fprintln(os.Stderr, "  ...")
+	fmt.Fprintln(os.Stderr)
+	fmt.Fprintln(os.Stderr, "Fields:")
+	fmt.Fprintln(os.Stderr)
+	fmt.Fprintln(os.Stderr, "  name:         the name of the metric (when grouped by name)")
+	fmt.Fprintln(os.Stderr, "  tag:          the tag key (when grouped by tag)")
+	fmt.Fprintln(os.Stderr, "  value:        the tag value (when grouped by tag)")
+	fmt.Fprintln(os.Stderr, "  count:        the number of metric points")
+	fmt.Fprintln(os.Stderr, "                the example above shows that 4002 metric points that had tag \"fruit\"=\"banana\" were duplicates")
+	fmt.Fprintln(os.Stderr, "  percentGroup: out of all of the metric points with the same name/tag (depending on grouping), what percentage of them were out-of-order/duplicates (depending on classification)?")
+	fmt.Fprintln(os.Stderr, "                the example above shows that ~4.301% of all metric points with name \"fruit.weight\" were out of order")
+	fmt.Fprintln(os.Stderr, "  percentClass: out of all of the metric points that were out-of-order/duplicates (depending on classification), what percentage of them had this name/tag (depending on grouping)?")
+	fmt.Fprintln(os.Stderr, "                the example above shows that ~2.063% of all metric points that were out of order had tag fruit=apple")
+	fmt.Fprintln(os.Stderr, "  percentTotal: out of all of the metric points, what percentage of them had this name/tag (depending on grouping) and were out-of-order/duplicates (depending on classification)?")
```

Comment:
I think it's best to word these as declarative statements rather than interrogative.